core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -400,7 +400,7 @@ oppia.controller('CreatorDashboard', [
             // the scope (the property cannot sit directly on the scope)
             // Reference https://stackoverflow.com/q/12618342
             $scope.suggestionData = {newSuggestionHtml: newContent.html};
-            $scope.showSuggestionEditor = false;
+            $scope.suggestionEditorIsShown = false;
```

Comment:
Note: This is a boolean. So, don't name it showSuggestionEditor, which suggests a method that, when executed, opens the suggestion editor. Instead use the naming convention here since you're checking the status of a property.